client/src/components/Reviews/components/AddReview.jsx

```diff
@@ -86,11 +81,12 @@ function AddReview({ handleModalToggle, allCharacteristics, productId }) {
     const submittedData = {
       ...reviewText,
       product_id: Number(productId),
-      rating,
+      rating: Number(rating),
       recommend: isRecommended,
       characteristics: characteristicSubmit,
       photos: [],
     };
+    console.log(submittedData);
```

Comment:
Do you want to keep this `console.log` or is it for debugging?

Reply:
i forgot about it, ty